docs/cvl/invariants.md

```diff
@@ -66,9 +66,12 @@ possible values of the parameters.
 
 While verifying an invariant, the Prover checks two things.  First, it checks
 that the invariant is established after calling any constructor.  Second, it checks
-that the invariant holds after the execution of any contract method, assuming
+that the invariant holds after the execution of any methods, assuming
 that it held before the method was executed (if it does hold, we say the method
-*preserves* the invariant).
+*preserves* the invariant). By default, an invariant fill be checked for all (non-view) method
+on all contracts in the scene - this can be further configured by {ref}`-parametric-contracts`.
+View methods are excluded from invariant checking as by definition they cannot change the state 
```

Comment:
```suggestion
`View` methods are excluded from invariant checking as by definition they cannot change the state 
```

Reply:
Should we also mention that `pure` methods are skipped? Technically they are view methods as well